scripts/OGC_tests/configure_sta10_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "Downloading ets-sta10 repo"
+
+git clone https://github.com/opengeospatial/ets-sta10.git
+cd ets-sta10
+
+mvn clean install 
+
+cd target
+
+cp ets-sta10-0.8-SNAPSHOT-ctl.zip /home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE/scripts
+cp ets-sta10-0.8-SNAPSHOT-deps.zip /home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE/resources/lib
+
+cd ../../
+
+cd /home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE/scripts
+unzip ets-sta10-0.8-SNAPSHOT-ctl.zip
+
+cd
+cd /home/travis/build/mozilla-sensorweb/sensorthings/TE_BASE/resources/lib
+unzip ets-sta10-0.8-SNAPSHOT-deps.zip
+
+cd ../../../
```

Comment:
There are a lot of `cd`s :)

After unifying all the scripts into a single one, I would save the output of `pwd` into a variable and would make all the `cd`s based on that.